types/node-telegram-bot-api/index.d.ts

```diff
@@ -40,42 +41,57 @@ declare namespace TelegramBot {
     type DocumentMimeType = 'application/pdf' | 'application/zip';
 
     type MessageType =
-        'text' |
-        'animation' |
-        'audio' |
-        'channel_chat_created' |
-        'contact' |
-        'delete_chat_photo' |
-        'document' |
-        'game' |
-        'group_chat_created' |
-        'invoice' |
-        'left_chat_member' |
-        'location' |
-        'migrate_from_chat_id' |
-        'migrate_to_chat_id' |
-        'new_chat_members' |
-        'new_chat_photo' |
-        'new_chat_title' |
-        'passport_data' |
-        'photo' |
-        'pinned_message' |
-        'sticker' |
-        'successful_payment' |
-        'supergroup_chat_created' |
-        'video' |
-        'video_note' |
-        'voice' |
-        'voice_chat_started' |
-        'voice_chat_ended' |
-        'voice_chat_participants_invited' |
-        'voice_chat_scheduled' |
-        'message_auto_delete_timer_changed' |
-        'chat_invite_link' |
-        'chat_member_updated';
-
-    type MessageEntityType = 'mention' | 'hashtag' | 'cashtag' | 'bot_command' | 'url' | 'email' | 'phone_number' |
-        'bold' | 'italic' | 'underline' | 'strikethrough' | 'code' | 'pre' | 'text_link' | 'text_mention' | 'spoiler';
+        | 'text'
+        | 'animation'
+        | 'audio'
+        | 'channel_chat_created'
+        | 'contact'
+        | 'delete_chat_photo'
+        | 'document'
+        | 'game'
+        | 'group_chat_created'
+        | 'invoice'
+        | 'left_chat_member'
+        | 'location'
+        | 'migrate_from_chat_id'
+        | 'migrate_to_chat_id'
+        | 'new_chat_members'
+        | 'new_chat_photo'
+        | 'new_chat_title'
+        | 'passport_data'
+        | 'photo'
+        | 'pinned_message'
+        | 'sticker'
+        | 'successful_payment'
+        | 'supergroup_chat_created'
+        | 'video'
+        | 'video_note'
+        | 'voice'
+        | 'video_chat_started'
+        | 'video_chat_ended'
+        | 'video_chat_participants_invited'
+        | 'video_chat_scheduled'
+        | 'message_auto_delete_timer_changed'
+        | 'chat_invite_link'
+        | 'chat_member_updated';
```

Comment:
This still needs the new [`web_app_data` message type](https://github.com/yagop/node-telegram-bot-api/compare/v0.56.0...72feca3d02293a3567ed927ce3792a33338da0eb#diff-a3e46f467e4f897823750e8a75b8e189d60e8ac5cc61ceaff63eac0e15f1c22fR58).

Reply:
Oh, thanks I missed it